validation/src/sobol_estimator_variance/convergence_stddev_old.py

```diff
@@ -0,0 +1,342 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+@author: Michaël Baudin (EDF), Antoine Dumas (Phiméca)
+"""
+
+
+from __future__ import print_function
+import openturns as ot
+from gsobollib import gsobolDistribution, gsobol
+from ishigamilib import (
+    ishigamiDistribution,
+    ishigamiGSymbolic,
+)
+from numpy import sqrt, array
+import numpy as np
+import matplotlib as mpl
+
```

Comment:
this one looks similar the new isnt it ? why keep it ?

Reply:
You are right: I remove it.